openapi.yml

```
@@ -244,6 +261,16 @@ paths:
               schema:
                 $ref: '#/components/schemas/AdministrationSearchResult'
 components:
+  responses:
+    403Forbidden:
+      description: Forbidden
+      <<: *problem+json-content
+    409Conflict:
+      description: Conflict
+      <<: *problem+json-content
+    500ErrorInternal:
```

```suggestion
    500InternalServerError:
```

if we use http message, then always do it.